modules/core/shared/src/test/scala/gem/math/JulianDateSpec.scala

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import cats.tests.CatsSuite
+import cats.{ Eq, Show }
+import gem.arb._
+
+import java.time.LocalDateTime
+
+
+@SuppressWarnings(Array("org.wartremover.warts.ToString", "org.wartremover.warts.Equals"))
+final class JulianDateSpec extends CatsSuite {
```

Comment:
Maybe we could add a test for `Eq` using `discipline`

Reply:
Agree, see [here](https://github.com/gemini-hlsw/gem/blob/master/modules/core/shared/src/test/scala/gem/math/AngleSpec.scala#L17) for an example; there's an equivalent for `Order` should we choose to upgrade.